src/App.tsx

```diff
@@ -10,7 +10,8 @@ import { fab } from "@fortawesome/free-brands-svg-icons";
 library.add(fas, far, fab);
 
 const App = () => {
-  const [todoList, setTodoList] = useState([]);
+  const [todoList, setTodoList]: [todoList: Todo[], setTodoList: any] =
+    useState([]);
```

Comment:
you need to know the shape of your todoList so you can declare the types

so your todolist is an array of objects with each object containing a title and an id, that means each todolist has a shape like this

```{
    title: string
    id: string
}
```

create a file outside the component folder and call it types.ts then inside of it add this

```
type TtodoItem = {
    title: string
    id: string
}

export type TtodoItemList = TtodoItem[]
```

then in your `App.tsx` import `TtodoItemList` and you can use it like this

```suggestion
        const [todoList, setTodoList] = useState<TtodoItemList>([]);
```